docs/dev/table/functions.md

```diff
@@ -1274,6 +1274,17 @@ ATAN2(numeric1, numeric2)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+COSH(numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the hyperbolic cosine of <code>NUMERIC</code>. Return value type is <code>DOUBLE</code>.</p>
```

Comment:
Hi @yanghua, please reformat the doc as usual. Besides, maybe we can move the return type description to a new line for now, like 

> Returns the hyperbolic cosine of \<i>NUMERIC\</i>. 
> The return type is \<i>DOUBLE\</i>.